databases/redis/Portfile

```diff
@@ -10,13 +10,13 @@ PortGroup           makefile 1.0
 legacysupport.newest_darwin_requires_legacy 15
 
 name                redis
-version             7.2.4
+version             7.4.0
 revision            0
 categories          databases
 platforms           darwin
-license             BSD
+license             {SSPL RSALv2}
```

Comment:
I don't think this license is known to MacPorts, and `port lint` errors. I doesn't seem to be defined in `distributable_lib.tcl` not sure how one would go about this. If it's actually a recognized SPDX license, the one could probably add it in MacPorts as well - @jmroot any advice here?

Reply:
I’m fine with dropping the other license too. We’re still telling users the correct usage requirements for the program even if we only mention one of the licenses.